Sources/Events/EventsManager.swift

```diff
@@ -114,8 +119,20 @@ actor EventsManager: EventsManagerType {
     }
     #endif
 
-    func flushEvents(batchSize: Int) async throws -> Int {
-        let featureEventsFlushed = try await self.flushFeatureEvents(batchSize: batchSize)
+    func flushEvents(batchSize: Int = defaultEventBatchSize) async throws -> Int {
```

Comment:
Since the `EventsManager` class is used through the `EventsManagerType` protocol, it currently doesn't allow to use the version of this method with no parameters `flushEvents()` because it's not declared in the `EventsManagerType` protocol.

If we're going to always use it with the default event batch size, then we could change the protocol declaration from
```swift
func flushEvents(batchSize: Int) async throws -> Int
```
to 
```swift
func flushEvents() async throws -> Int
```